ale_linters/clojure/clj_kondo.vim

```diff
@@ -1,6 +1,15 @@
 " Author: Masashi Iizuka <liquidz.uo@gmail.com>
 " Description: linter for clojure using clj-kondo https://github.com/borkdude/clj-kondo
 
+function! ale_linters#clojure#clj_kondo#GetCommand(buffer) abort
+    let l:conf_dir = ale#path#FindNearestDirectory(a:buffer, '.clj-kondoo')
+    let l:conf_option = empty(l:conf_dir)
+          \ ? ''
```

Comment:
The `\` should come before the 4 spaces, like in lines 28-34.

Reply:
@RageKnify Thanks! I fixed it.